src/OpalCompiler-Core/Behavior.extension.st

```diff
@@ -104,3 +104,29 @@ Behavior >> sourceCodeTemplate [
 	| temporary variable names |
 	statements'
 ]
+
+{ #category : #'*OpalCompiler-Core' }
+Behavior >> sourceCodeTemplateFor: aClassOrMetaclass [ 
+	"Answer an expression to be edited and evaluated in order to define 
+	methods in this class or trait."
+
+		^ aClassOrMetaclass isMeta 
```

Comment:
it should use isClassSide, or better, use polymorphism instead of if

Reply:
I did try polymorphism first, but it felt awkward piling another extra method onto Object.  Whether its class-side or instance-side, both seems to be a function of Behaviour, so in this case I went with the conditional.
I'll update it to use isClassSide.

Reply:
I've just tested it with isClass, and that doesn't work.
I've added OCSourceTemplates>>testsourceCodeTemplateFor to help demonstrate this.